sdk/communication/azure-communication-phonenumbers/samples/update_phone_number_capabilities_sample.py

```diff
@@ -26,15 +26,15 @@
 
 connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
 phone_number_to_update = os.getenv(
-    "AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER_TO_UPDATE" # e.g. "+15551234567"
+    "AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER" # e.g. "+15551234567"
 ) 
 phone_numbers_client = PhoneNumbersClient.from_connection_string(connection_str)
 
 def update_phone_number_capabilities():
     poller = phone_numbers_client.begin_update_phone_number_capabilities(
         phone_number_to_update,
-        PhoneNumberCapabilityType.INBOUND_OUTBOUND,
```

Comment:
Is this intended?

Reply:
You're totally right, I had it changed for testing purposes